Standard.AI.OpenAI/Services/Foundations/AIModels/AIModelService.cs

```diff
@@ -34,6 +35,15 @@ internal partial class AIModelService : IAIModelService
             return externalAIModelsResult.AIModels.Select(ConvertToAIModel).ToArray();
         });
 
+        public ValueTask<AIModel> RetrieveAIModelByIdAsync(string aiModelName) =>
+        TryCatch(async () =>
+        {
+            ExternalAIModel externalAIModel =
```

Comment:
```suggestion
        public ValueTask<AIModel> RetrieveAIModelByIdAsync(string aiModelId) =>
        TryCatch(async () =>
        {
            ExternalAIModel externalAIModel =
```

Here the aiModelId needs to be validated before the request is made and throw a InvalidAIModelException if the id is invalid.

Reply:
@Gummimundur not sure if I should title the exception 'InvalidAIModelException' or 'InvalidAIModelNameException'... thoughts?

Reply:
Resolving because this was added but re-open or leave comment for name change.